### DISCUSSION TITLE
Add param to edit an autocompletion completions sort

### DISCUSSION THREAD

**Discussion by badger-py** [microsoft/pylance-release#2845] [Enhancement]
Hi!
As default, autocompletion sort is **alphabetical**. But I such, it bad. Make some setting to set **the sort order** to **"most used"**.

**Comment by luabud** [comment]
Thank you for the feature request! I'm transferring this to a discussion for comments and upvotes. In the mean time, have you tried using the [IntelliCode](https://marketplace.visualstudio.com/items?itemName=VisualStudioExptTeam.vscodeintellicode) extension? It shows the autocompletion list sorted based on insights it takes from your code context combined with machine learning models. Here's an example of it working: https://docs.microsoft.com/en-us/visualstudio/intellicode/media/python-intellicode.gif
